scripts/hooks/pre-commit.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# lint modified go files
+golangci-lint run --fix --new -c .golangci.yml
```

Comment:
Are we sure it will use the correct `.golangci.yml` file?

Reply:
Yep, after I changed it to `../.golangci.yml` or `.golangci.yaml`, it will failed

![image](https://github.com/cosmos/cosmos-sdk/assets/150114626/d30d5264-c4bc-4d82-bba8-11643b42cd22)